instrumentation/undertow/undertow-1.4/src/test/java/io/opentelemetry/javaagent/instrumentation/hypertrace/undertow/v1_4/UndertowInstrumentationTest.java

```diff
@@ -108,8 +108,8 @@ void postUrlEncoded() throws InterruptedException, TimeoutException, IOException
       assertEquals(200, response.code());
     }
 
-    List<List<SpanData>> traces = TEST_WRITER.getTraces();
```

Comment:
@ryandens another comment on using `final` everywhere. The test class looks messy, I am obviously against using `final` where no necessary, but here it's even messier bc it's not used consistently. It degrades code quality.